bootloader/RAK4630/README.md

```diff
@@ -201,3 +202,42 @@ Steps as below:
 - Select Target->Production Programming. 
 
 <img src="assets/bootloader-segger.png" width="80%" caption="SWD connection" />
+
+## Update over RAKDAP1
+
+### Hardware and Tools Required
+
+Required:
+
+- WisBlock Base RAK5005-O
+- WisBlock Core RAK4631
+- The latest bootloader (V0.4.2 - WisCore_RAK4631_Board_Bootloader.zip)
+- [RAKDAP1 tool](https://store.rakwireless.com/products/daplink-tool)
+- pyOCD
+
+### Install pyOCD
+
+pyOCD is an open source Python package for programming and debugging microcontrollers. Refer to [Quick Start Guide](https://docs.rakwireless.com/Product-Categories/Accessories/RAKDAP1-Flash-and-Debug-Tool/Quickstart/#installation) for detailed instructions on how to install the pyOCD tool.
+
+Install the RAK4631 module support package, using the following command:
+```c
+pyocd pack --install nrf52
+```
+### Connect RAKDAP1 adapter to SWD of the module
+
+Use the figure below as a guide to connect RAKDAP1 SWD to RAK4631.
+
+<img src="assets/rak4600-rakdap1.svg" width="80%" caption="RAKDAP1 connection" />
```

Comment:
@mtuxpe 
Wrong image. 
You show a RAK4600, but this is for the RAK4631

Reply:
@beegee-tokyo The new image shows only RAK4631and RAKDAP1 . I am unable to build a full image (RAK4631 + RAK4005-O)